### DISCUSSION TITLE
Order Book / Market Depth panel

### DISCUSSION THREAD

**Discussion by sylr** [grafana/grafana#59763] [Dashboards, Panels & Visualizations]
Hi :wave:

To complement the recent OHLC candelstick panel It would be really cool if grafana could propose an order book/market depth visualization panel such as:

![](https://storage.googleapis.com/cgyinsights/wp-content/2016/10/Pasted-image-at-2016_10_10-02_25-PM.png)

**Comment by leeoniya** [comment]
certainly considered it when writing the candlestick panel :)

wonder if this panel would include the table or just rely on an addl user-added table panel below it (with all the options that already has). 🤔

**Reply by sylr** [reply]
I think it would be nice to have the table in the same panel to avoid having to duplicate calls to get the same data. However, I think it should be optional like the legend of the time series panel.

**Reply by leeoniya** [reply]
you can reuse the data from another panel using the built-in Dashboard datasource, so that shouldnt be an issue. and it being "optional" is exactly the type of custom settings you would avoid by keeping them decoupled.

**Reply by sylr** [reply]
Then the graph only would certainly suffice! :+1:

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
